LocalWiki.php

```diff
@@ -380,7 +402,39 @@ function onBeforePageDisplay( OutputPage $outputPage ) {
 				]
 			]
 		];
-
+		
+		// SocialProfile/UserStats configurations
+		require_once( "$IP/extensions/SocialProfile/UserStats/EditCount.php" );
+			// User level definitions
+			$wgUserLevels = [
+				'Lớp lá' => 0,
+				'Mầm non' => 1000,
+				'Lớp 1' => 1500,
+				'Lớp 2' => 2000,
+				'Lớp 3' => 3000,
+				'Lớp 4' => 4000,
+				'Lớp 5' => 5000,
+				'Lớp 6' => 6500,
+				'Lớp 7' => 7500,
+				'Lớp 8' => 8500,
+				'Lớp 9' => 9500,
+				'Lớp 10' => 11000,
+				'Lớp 11' => 13000,
+				'Lớp 12' => 15000,
+				'Đại học' => 50000,
+				'Cao học' => 100000
+			];
+			// User point for each actions and the namespaces that counts toward
+			$wgUserStatsPointValues['edit'] = 50;
```

Comment:
Should be in LocalSettings.php — https://github.com/miraheze/mw-config/blob/f487bc3993e5cf5ebdb93931123d5f7357cdb50d/LocalSettings.php#L5555-L5560